CONTRIBUTING.md

```diff
@@ -90,12 +90,70 @@ Follow GitHub's fork-branch-pull request pattern.
 7. Provide your full name or GitHub username and your company or organization if applicable.
 
 #### Once a Pull Request Ready for Review
-1. Verify the commit message and PR title use the template `Fix #XYZ, SHORT_DESCRIPTION`.
-2. Verify there is only one commit message. Squash or amend the commit messages if necessary.
+1. Verify the commit message and PR title use the template `Fix #XYZ, SHORT_DESCRIPTION`. The commit message can use `Fix #XYZ, LONG_DESCRIPTION` if necessary. 
+2. Verify there is one commit message per topic. For example, review the [provided pull request](https://github.com/nasa/cFE/pull/1203/commits). 
+3. Squash or amend the commit messages if necessary. For more information, review the sections [How to Squash Commits](#how-to-squash-commits) and [How to Amend Commits](#how-to-amend-commits) .  
 3. Verify that the PR passes all checks.
-4. The project team will label the issue and evaluate the pull request in the weekly configuration control board (CCB) meeting. For more information, visit [The cFS CCB Process.](https://github.com/nasa/cFS/wiki/The-cFS-CCB-Process)
+4. The project team will label the issue and evaluate the pull request in the weekly configuration control board (CCB) meeting. For more information, visit [The cFS CCB Process](https://github.com/nasa/cFS/wiki/The-cFS-CCB-Process).
 5. If the pull request is accepted, it will be merged into cFS.
 
+#### How to Squash Commits
```

Comment:
This seems to include both a rebase and a squash... do you want to address these separately?  There's also a few different methods, see https://github.com/nasa/cFS-GroundSystem/pull/137 (comments near the end from Joe and I).

Reply:
Created three sections describing the three methods on how to squash commits. The sections are Interactive Rebase, Soft Reset, and Replace Branch.